confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfig.java

```diff
@@ -220,8 +220,8 @@ public class ConfigNodeConfig {
   private long dataRegionRatisPreserveLogsWhenPurge = 1000;
 
   /* first election timeout shares between 3 regions */
-  private long ratisFirstElectionTimeoutMinMs = 50;
-  private long ratisFirstElectionTimeoutMaxMs = 150;
+  private long ratisFirstElectionTimeoutMinMs = 2000;
```

Comment:
whether is  this time a better,  can our election-time greater than this time?

Reply:
Yes. First election time is only a special case for election time. Currently there are bugs triggered by first election, so it's better to increase first election time to a matter of seconds.